service/controller/legacy/v27/version_bundle.go

```diff
@@ -8,8 +8,8 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "TODO",
-				Description: "TODO",
+				Component:   "encrypter",
+				Description: "Fixes for AWS China.",
```

Comment:
Let's be a bit more specific. It was something like fixing Vault authentication / race condition.